src/Elasticsearch.Net/Connection/HttpConnection.cs

```diff
@@ -66,7 +66,6 @@ protected virtual HttpWebRequest CreateWebRequest(RequestData requestData)
 			if (requestData.HttpCompression)
 			{
 				request.AutomaticDecompression = DecompressionMethods.GZip | DecompressionMethods.Deflate;
-				request.Headers.Add("Accept-Encoding", "gzip,deflate");

```

Comment:
Did you mean to delete this too?

Reply:
Yes, `request.AutomaticDecompression = DecompressionMethods.GZip | DecompressionMethods.Deflate;` sets the `"Accept-Encoding"` header

Reply:
:+1: